### ISSUE TITLE
fails to transform img and mask at the same time

### ISSUE THREAD

**Issue by Chen-yu-Zheng** [albumentations-team/albumentations#1023]
## 🐛 Bug

It seems that it fails to transform img and mask at the same time

## To Reproduce
This is my code

import torch
import cv2
from utils.rle2img import *
import torchvision.transforms as T
from torch.utils import data as D
import configs
import albumentations as A
import pandas as pd

class TianChiDataset(D.Dataset):
    def __init__(self, paths, rles, test_mode=False):
        self.paths = paths
        self.rles = rles
        self.test_mode = test_mode
        
        self.len = len(paths)

        self.transform = A.Compose([
            A.Resize(configs.IMAGE_SIZE, configs.IMAGE_SIZE),
            A.HorizontalFlip(p=1),
            A.VerticalFlip(p=1),
            A.RandomRotate90(p=1)
        ])

        self.transform_img = T.Compose([
            T.ToTensor(),
            T.Normalize([0.40464398, 0.42690134, 0.39236653],
                        [0.20213476, 0.18353915, 0.17596193])
        ])
        self.transform_mask = torch.tensor
        
    # get data operation
    def __getitem__(self, index):
        img = cv2.imread(self.paths[index])
        #cv2 导入的是BGR形式
        img = cv2.cvtColor(img, cv2.COLOR_BGR2RGB)

        if not self.test_mode:
            mask = rle_decode(self.rles[index])
            augments = self.transform(image=img, mask=mask)

            img = self.transform_img(augments['image'])
            mask = self.transform_mask(augments['mask']).float()
            return img, mask
        else:
            return self.transform_img(img), ''        
    
    def __len__(self):
        """
        Total number of samples in the dataset
        """
        return self.len
    

def get_TianchiDataset():
    train_mask = pd.read_csv(configs.ROOT, sep='\t', names=['name', 'mask'])
    train_mask['name'] = train_mask['name'].apply(lambda x: 'data/train/' + x)
    train_mask['mask'] = train_mask['mask'].fillna('')

    dataset = TianChiDataset(
        train_mask['name'].values,
        train_mask['mask'].values,      
        False
    )
    return dataset

## Expected behavior
when I add 
            A.HorizontalFlip(p=1),
            A.VerticalFlip(p=1),
            A.RandomRotate90(p=1)
mask and img didn't be transformed at the same time

## Environment

 - Albumentations version (e.g., 0.1.8): 1.0.3
 - Python version (e.g., 3.7): 3.8.11
 - OS (e.g., Linux): Win
 - How you installed albumentations (`conda`, `pip`, source): pip

**Comment by BloodAxe** [comment]
Hi! From time to time we see similar bug reports that claims there is inconsistency between image & mask augmentations. However, upon detailed look it happens to be a bug on user side in 99% cases. For all augmentations we have, there are unit tests that verify both image & mask transformed identically.

Likewise, in your case you instantiate `TianChiDataset` using `test_mode=False`. This means albumentations pipeline is not applied at all to your images (See `if not self.test_mode:`). So I don't see how reported issue relates to albumentations. Please provide a minimalistic example to reproduce the issue (Ideally in Colab or self-contained python snippet) and reopen the ticket if the problem persists.
